docs/customization/form.rst

```diff
@@ -123,6 +123,86 @@ Need more information?
 
     Some of the forms already have extensions in Sylius. Learn more about Extensions `here <http://symfony.com/doc/current/bundles/extension.html>`_.
 
+For instance the ``ProductVariant`` admin form is defined under ``Sylius/Bundle/ProductBundle/Form/Type/ProductVariantType.php`` and later extended in
+``Sylius/Bundle/CoreBundle/Form/Extension/ProductVariantTypeExtension.php``. If you again extend the base type form like this:
+
+.. code-block:: yaml
+
+    services:
+        app.form.extension.type.product_variant:
+            class: AppBundle\Form\Extension\ProductVariantTypeMyExtension
+            tags:
+                - { name: form.type_extension, extended_type: Sylius\Bundle\ProductBundle\Form\Type\ProductVariantType, priority: -5 }
+
+your form extension will also be executed. Whether before or after the other extensions depends on priority tag set.
```

Comment:
We could add an information about which priorities are executed after and before such a form